velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -37,7 +39,99 @@ MATCHER(IsInf, "is Infinity") {
   return arg && std::isinf(*arg);
 }
 
-class ProbabilityTest : public functions::test::FunctionBaseTest {};
+class ProbabilityTest : public functions::test::FunctionBaseTest {
+ protected:
+  template <typename ValueType>
+  auto poissonCDF(
+      const std::optional<double>& lambda,
+      const std::optional<ValueType>& value) {
+    return evaluateOnce<double>("poisson_cdf(c0, c1)", lambda, value);
+  }
+
+  template <typename ValueType>
+  auto binomialCDF(
+      std::optional<ValueType> numberOfTrials,
+      std::optional<double> successProbability,
+      std::optional<ValueType> value) {
+    return evaluateOnce<double>(
+        "binomial_cdf(c0, c1, c2)", numberOfTrials, successProbability, value);
+  }
+
+  template <typename ValueType>
+  void runPoissonCDFTests() {
```

Comment:
This method names can be changed to just poissonCDFTests() and binomialCDFTests()